app/forms/base_form.rb

```diff
@@ -42,7 +42,7 @@ def save
 
     # List of form attributes not to be passed to model
     def exclude_from_model
-      []
+      %i[lookup_postcode lookup_error]
```

Comment:
This class (`BaseForm`) holds the default configuration for that method. If you want a specific configuration for your class, you should be overriding this in the class that requires it (`AddressForm`).

Reply:
changed👍